x/accesscontrol/types/message_dependency_mapping.go

```diff
@@ -1,10 +1,22 @@
 package types
 
 import (
+	fmt "fmt"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/gogo/protobuf/proto"
 )
 
+var ErrNoCommitAccessOp = fmt.Errorf("message dependency mapping AccessOps don't terminate with COMMIT access op")
```

Comment:
Thoughts on this message? 

```suggestion
var ErrNoCommitAccessOp = fmt.Errorf("MessageDependencyMapping doesn't terminate with AccessType_COMMIT")
```